drivers/bmx280/bmx280_saul.c

```diff
@@ -35,9 +35,9 @@ static int read_temperature(const void *dev, phydat_t *res)
 
 static int read_pressure(const void *dev, phydat_t *res)
 {
-    res->val[0] = bmx280_read_pressure((bmx280_t *)dev) / 100;
+    res->val[0] = bmx280_read_pressure((bmx280_t *)dev) / 10;
```

Comment:
I think it would be worth a comment regarding the overflow safety of this operation, as theoretically it shouldn't give pressures higher than 1100 deca-pascals

Reply:
Even better would be using [`phydat_fit`](https://api.riot-os.org/group__sys__phydat.html#gafafe8717882db85c250f203b020f8863). This would only loose precision if the value otherwise doesn't fit.

Reply:
Updated to use `phydat_fit`